scene/2d/gpu_particles_2d.h

```diff
@@ -58,6 +58,7 @@ class GPUParticles2D : public Node2D {
 	bool local_coords;
 	int fixed_fps;
 	bool fractional_delta;
+	bool show_visibility_rect;
```

Comment:
This should be in `#ifdef TOOLS_ENABLED` too.

Reply:
thanks, done!